app/client/widgets/AttachmentsEditor.ts

```diff
@@ -97,6 +99,7 @@ export class AttachmentsEditor extends NewBaseEditor {
       const index = use(this._index);
       return index === null ? null : use(this._attachments)[index];
     }));
+    this._isUploading = Observable.create(null, false);
```

Comment:
`this._isUploading = Observable.create(this, false);` ? Or ensure it is disposed at some point?